types/oxon/buttonboard/meta.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Buttonboard LoRaWan",
+  "name": "Buttonboard (LoRaWAN)",
```

Comment:
please try avoiding changing names as it will create a new device type for it in the db